### ISSUE TITLE
Dropdowns are broken with CEF v127

### ISSUE THREAD

**Issue by Hardy535** [altmp/altv-issues#2395]
### Description of the problem

Dropdowns are not correctly displayed since CEF version v127.
They always appear in the top left corner and not directly below the dropdown element.
This seems to only be a "rendering" issue as hovering over the items where they normally are works.

https://gyazo.com/e41536135851d03dd2d5ef287dcc5bf0

### Reproduction steps

Create a dropdown menu in CEF and use it.

```html
<select name="cars" id="cars">
  <option value="volvo">Volvo</option>
  <option value="saab">Saab</option>
  <option value="mercedes">Mercedes</option>
  <option value="audi">Audi</option>
</select>
```

### Expected behaviour

Should work as before.

### Operating system

Windows 23H2

### Version

16.2.37

### Confirmation of issue's presence

- [X] *By submitting this ticket, I affirm that I have verified the presence of this issue on the latest RC (Release Candidate) version available at the time of writing this ticket.*

**Comment by Lekenthu** [comment]
I confirm.

**Comment by Doxoh** [comment]
fixed in `16.2.39-rc.1`

**Comment by Hardy535** [comment]
There seems to be another issue with the dropdowns since CEF v127.
Not only were the positions not correct as mentioned above, some users are reporting other weird rendering issues with them.

I can't reproduce it myself but other players are reporting it:

When using the dropdown, not a list appears but another (old) webview.
In this case it's a webview I show when the player logs in for discord authentication, however this webview is **destroyed** after the playerr logged in.
It seems to be some weird artifact the the users are seeing here as of the time the users see this, this "artifact" of a webview doesn't even exist anymore.

For most users it seems fine, but not for everyone.
We couldn't narrow down why this issue happens, doesn't seem specific to Intel/AMD/NVIDIA.
Players tried reinstalling alt:V but with no luck, before the CEF update this was not an issue.

Some clips from players reporting this problem:

https://medal.tv/games/gta-v/clips/jp5ngkRVbcBrrroXP?invite=cr-MSxYdWUsMjMyNDk3MzE1LA
https://gyazo.com/604e054bf89742ea30cda0c10de7f738
